src/utils/helper.js

```diff
@@ -457,15 +455,15 @@ export const updateInternalQuery = (
 	}
 };
 // extracts query options from defaultQuery if set
-export const extractQueryFromDefaultQuery = (defaultQuery) => {
+export const extractQueryFromDefaultQuery = (defaultQuery, props) => {
 	let queryToBeReturned = {};
 	if (defaultQuery) {
 		const evaluateQuery = defaultQuery();
```

Comment:
@ShahAnuj2610 Make the code readable, it's hard to understand how you're passing `defaultQuery`.